api/src/main/java/org/openmrs/module/spa/Item.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.spa;
+
+import org.openmrs.BaseOpenmrsData;
+import org.openmrs.User;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+/**
+ * Please note that a corresponding table schema must be created in liquibase.xml.
+ */
+//Uncomment 2 lines below if you want to make the Item class persistable, see also FrontendSPADaoTest and liquibase.xml
+//@Entity(name = "spa.Item")
+//@Table(name = "spa_item")
+public class Item extends BaseOpenmrsData {
```

Comment:
What's all this for? Maybe I'm lacking some basic OpenMRS knowledge here, but maybe it could be briefly explained in the class documentation?

Reply:
Oh wait, I imagine this is OpenMRS module boilerplate... don't worry about it.